--- share/libtool/configure_old.ac	2024-04-09 14:05:54.288456000 +0200
+++ share/libtool/configure.ac	2024-04-09 14:19:03.272899019 +0200
@@ -34,7 +34,7 @@
 # project.  Adding LT_WITH_LTDL to your project configure.ac will then
 # configure this directory if your user doesn't want to use the installed
 # libltdl.
-
+m4_pattern_allow([AM_INIT_AUTOMAKE])
 AC_PREREQ(2.59)dnl We use AS_HELP_STRING
 
 
@@ -51,7 +51,6 @@
 ## ------------------------ ##
 ## Automake Initialisation. ##
 ## ------------------------ ##
-
 AM_INIT_AUTOMAKE([gnu subdir-objects])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
