--- share/autoconf/autom4te_old.cfg	2024-03-03 10:03:02.000000000 +0100
+++ share/autoconf/autom4te.cfg	2024-04-09 13:29:16.243818738 +0200
@@ -115,9 +115,9 @@
 # This intermediate language is used by aclocal to build aclocal.m4.
 
 begin-language: "Autoconf-without-aclocal-m4"
-args: --prepend-include '/builds/buildroot.org/toolchains-builder/build/armv7-eabihf--musl--stable-2024.02-1/share/autoconf'
+args: --prepend-include '/tmp/bazel-working-directory/onnxruntime-arm-musleabihf-cross/external/arm32_gcc_linux_x86_64_musl/share/autoconf'
 args: --cache=autom4te.cache
-args: autoconf/autoconf.m4f
+args: /tmp/bazel-working-directory/onnxruntime-arm-musleabihf-cross/external/arm32_gcc_linux_x86_64_musl/share/autoconf/autoconf/autoconf.m4f
 args: acsite.m4?
 args: --mode 777
 args: --language M4sh
@@ -142,8 +142,8 @@
 ## -------- ##
 
 begin-language: "Autotest"
-args: --prepend-include '/builds/buildroot.org/toolchains-builder/build/armv7-eabihf--musl--stable-2024.02-1/share/autoconf'
-args: autotest/autotest.m4f
+args: --prepend-include '/tmp/bazel-working-directory/onnxruntime-arm-musleabihf-cross/external/arm32_gcc_linux_x86_64_musl/share/autoconf'
+args: /tmp/bazel-working-directory/onnxruntime-arm-musleabihf-cross/external/arm32_gcc_linux_x86_64_musl/share/autoconf/autotest/autotest.m4f
 args: package.m4?
 args: local.at?
 args: --mode 777
@@ -156,8 +156,8 @@
 ## ---- ##
 
 begin-language: "M4sh"
-args: --prepend-include '/builds/buildroot.org/toolchains-builder/build/armv7-eabihf--musl--stable-2024.02-1/share/autoconf'
-args: m4sugar/m4sh.m4f
+args: --prepend-include '/tmp/bazel-working-directory/onnxruntime-arm-musleabihf-cross/external/arm32_gcc_linux_x86_64_musl/share/autoconf'
+args: /tmp/bazel-working-directory/onnxruntime-arm-musleabihf-cross/external/arm32_gcc_linux_x86_64_musl/share/autoconf/m4sugar/m4sh.m4f
 args: --mode 777
 args: --language M4sugar
 end-language: "M4sh"
@@ -168,6 +168,6 @@
 ## ------- ##
 
 begin-language: "M4sugar"
-args: --prepend-include '/builds/buildroot.org/toolchains-builder/build/armv7-eabihf--musl--stable-2024.02-1/share/autoconf'
-args: m4sugar/m4sugar.m4f
+args: --prepend-include '/tmp/bazel-working-directory/onnxruntime-arm-musleabihf-cross/external/arm32_gcc_linux_x86_64_musl/share/autoconf'
+args: /tmp/bazel-working-directory/onnxruntime-arm-musleabihf-cross/external/arm32_gcc_linux_x86_64_musl/share/autoconf/m4sugar/m4sugar.m4f
 end-language: "M4sugar"
