--- bin/libtoolize_old	2024-04-05 11:48:17.819108000 +0200
+++ bin/libtoolize	2024-04-05 11:52:50.743506632 +0200
@@ -4300,7 +4300,7 @@
   if test -n "$_lt_pkgdatadir"; then
     pkgauxdir=$_lt_pkgdatadir/build-aux
     pkgltdldir=$_lt_pkgdatadir/libltdl
-    aclocaldir=$_lt_pkgdatadir/m4
+    aclocaldir=$_lt_pkgdatadir/../aclocal
   fi
   func_nonemptydir_p pkgauxdir
   func_nonemptydir_p pkgltdldir
